cmd/immuadmin/command/database.go

```diff
@@ -52,6 +52,8 @@ func addDbUpdateFlags(c *cobra.Command) {
 	c.Flags().Uint32("write-buffer-size", store.DefaultWriteBufferSize, "set the size of in-memory buffers for file abstractions")
 	c.Flags().Uint32("read-tx-pool-size", database.DefaultReadTxPoolSize, "set transaction read pool size (used for reading transaction objects)")
 	c.Flags().Bool("autoload", true, "enable database autoloading")
+	c.Flags().Duration("retention-period", 0, "duration of time to retain data in storage")
```

Comment:
a default constant might also be used when declaring `retention-period` flag?

Reply:
actually I am using the condition, > 0 to determine if to start the truncation process. Because truncation by default is not enabled. DO you think we need to add another flag, like `--enable-truncation` and then take in these params?

Reply:
no, I think it's ok